src/Cli.Tests/EndToEndTests.cs

```diff
@@ -18,7 +18,12 @@ public static void Setup(TestContext context)
     {
         if (!File.Exists(TEST_SCHEMA_FILE))
         {
-            File.Create(TEST_SCHEMA_FILE);
+            File.Create(TEST_SCHEMA_FILE).Dispose();
+        }
+
+        if (File.Exists(TEST_RUNTIME_CONFIG_FILE))
+        {
+            File.Delete(TEST_RUNTIME_CONFIG_FILE);
```

Comment:
Could this be simplified given File.Delete behavior? also, is it desireable to create the file during every classInitialize because pre-existing file may lead to unintended side effects?